Source/JavaScriptCore/yarr/YarrParser.h

```diff
@@ -1360,7 +1360,7 @@ class Parser {
             if (ch == errorCodePoint)
                 return;
 
-            classSetConstructor.atomPatternCharacter(static_cast<char32_t>(ch));
+            classSetConstructor.atomPatternCharacter(static_cast<char32_t>(ch), true);
```

Comment:
You should add comment like, `/* hyphenIsRange */ true`

Reply:
fixed